src/index.js

```diff
@@ -1,31 +1,27 @@
 
-var absolute = require('absolute');
-var assert = require('assert');
-var clone = require('clone');
-var fs = require('co-fs-extra');
-var is = require('is');
-var matter = require('gray-matter');
-var Mode = require('stat-mode');
-var path = require('path');
-var readdir = require('recursive-readdir');
-var rm = require('rimraf');
-var thunkify = require('thunkify');
-var unyield = require('unyield');
-var utf8 = require('is-utf8');
-var Ware = require('ware');
+import "babel-polyfill";
+
+import absolute from 'absolute';
+import assert from 'assert';
+import clone from 'clone';
+import fs from 'co-fs-extra';
+import is from 'is';
+import matter from 'gray-matter';
+import Mode from 'stat-mode';
+import path from 'path';
+import readdir from 'recursive-readdir';
+import rm from 'rimraf';
+import thunkify from 'thunkify';
+import unyield from 'unyield';
+import utf8 from 'is-utf8';
+import Ware from 'ware';
 
 /**
  * Thunks.
  */
 
-readdir = thunkify(readdir);
-rm = thunkify(rm);
-
-/**
- * Export `Metalsmith`.
- */
-
-module.exports = Metalsmith;
+const readdirt = thunkify(readdir);
+const rmt = thunkify(rm);
```

Comment:
Interesting. So imported values can't be reassigned at runtime? I didn't realize that...